tools/build-tools/src/repoPolicyCheck/handlers/assertShortCode.ts

```diff
@@ -101,16 +123,17 @@ export const handler: Handler = {
             for(const msg of getAssertMessageParams(s)){
                 // here we only want to looks at those messages that are not numbers,
                 // as we validated existing short codes above
-                if(msg.getKind() !== SyntaxKind.NumericLiteral){
+                if(!(msg instanceof NumericLiteral)){
```

Comment:
Let's avoid using `instanceof`.  It's probably ok in this context, but in general, using `instanceof` to detect type is not reliable in javascript

Reply:
Ok. Here I did it mostly for the type guard, so inside I didn't need to assert the other possible types inside the block in order to use their methods which weren't in a `NumericLiteral`. But point taken 👍 .